src/uct/ib/rc/accel/rc_mlx5_iface.c

```diff
@@ -517,10 +515,28 @@ UCS_CLASS_INIT_FUNC(uct_rc_mlx5_iface_common_t,
         goto cleanup_tm;
     }
 
-    self->super.config.fence       = uct_ib_device_has_pci_atomics(dev);
+    self->super.config.fence       = rc_config->fence != UCT_RC_FENCE_NONE;
```

Comment:
why we don't consider PCI atomics anymore for atomic fence flag?
does it mean that all atomic ops will have weak fence by default, before it was without any fence?